torch_brain/nn/rotary_attention.py

```diff
@@ -65,6 +71,10 @@ def __init__(
         self.to_kv = nn.Linear(context_dim, inner_dim * 2, bias=False)
         self.to_out = nn.Linear(inner_dim, dim)
 
+        self.use_efficient = os.getenv(
+            "TORCH_BRAIN_USE_EFFICIENT_ATTENTION", "true"
+        ).lower() in ("true", "1", "yes")
+
```

Comment:
I'm a little bothered by the vagueness of the phrase "efficient". Perhaps:
```suggestion
        self.use_xformers = os.getenv(
            "TORCH_BRAIN_USE_XFORMERS", "true"
        ).lower() in ("true", "1", "yes")
```